plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -507,6 +510,46 @@ public void testRequiresQueryPartitionFilterWithUppercaseColumnName()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAppendOnly()
+            throws Exception
+    {
+        String tableName = "test_append_only_" + randomNameSuffix();
+        Path tableLocation = Files.createTempFile(tableName, null);
+        copyDirectoryContents(new File(Resources.getResource("deltalake/append_only").toURI()).toPath(), tableLocation);
```

Comment:
nit: can we have a method that accepts `"deltalake/append_only", tableLocation` as params? this call is not readable

Reply:
I would handle these comments in a follow-up because the current logic is consistent with existing tests.

Reply:
Ok, let’s improve it please